code/modules/antagonists/traitor/equipment/Malf_Modules.dm

```diff
@@ -573,6 +575,11 @@ GLOBAL_LIST_INIT(blacklisted_malf_machines, typecacheof(list(
 	uses = 2
 	linked_ability_type = /obj/effect/proc_holder/ranged_ai/overload_machine
 
+/datum/action/innate/ai/ranged/overload_machine/New()
+	..()
+	desc = "[desc] There are [uses] reactivations remaining."
+	button.desc = desc
```

Comment:
This should be handled in the parent type, inside of `/update_desc` by appending the number of uses to the examine message if the initial(uses) is different from 1. Then when you update the uses count, call update_appearance(UPDATE_DESC) instead

Reply:
Alright so, I made the requested changes, I think. I'm not super proficient yet, but I added a description of my reasoning to the commit, so let me know if I did anything wrong!